src/course/syllabus/application/tribute-page/resources.md

```diff
@@ -0,0 +1,35 @@
+## Getting started
+
+- [Setting up a web page](https://oliverjam.es/blog/complete-guide-to-making-web-pages/)
+- [freeCodeCamp: Responsive web design](https://www.freecodecamp.org/learn/2022/responsive-web-design/)
+  > Note: sections of freeCodeCamp's curriculum used to be a part of our application requirements. We replaced it with our own projects. Around the same time, freeCodeCamp published a new project-based Responsive Web Design curriculum. It doesn't touch much on JavaScript and completing it all would take you a reasonable while, it will give you a fantastic foundation in building websites.
```

Comment:
"It doesn't touch much on JavaScript and completing it all would take you a reasonable while, but it will give you a fantastic foundation in building websites." (sorry to be a grammar nitpicker)